native/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Should these configs have our standard license header???

Reply:
The ones in the root of the folder don't have it, so I matched that here. Not to say that we shouldn't have them though.

Maybe we should decide amongst ourselves which files need to have it and then any changes to be made can be done in another PR to keep it separate?

Reply:
👍

Reply:
json documents don't support comments ;)

Reply:
This is why NPM needs support for a `package.js` file!